### PR TITLE
Small Bugs Fixed

### PR DESCRIPTION
**Pull request by Srijan221**
Small bugs arrised due to memory error or early stopping fixed.

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -25,12 +25,12 @@
 random.seed(seed)
 np.random.seed(seed)
 
-
+# Training of model
```
Comment (likith012) [inline train.py]: Providing comments at the top of the function is not a standard way, it's written at the start of the file. Refer to my other scripts. Refer to this tutorial https://realpython.com/documenting-python-code/

File: train.py [hunk 2]
```diff
@@ -152,6 +154,12 @@ def train(
 if __name__ == "__main__":
     """Main function to run the training of the model."""
 
+    # Set the GPU to allocate only used memory at runtime.
```
Comment (likith012) [inline train.py:156]: Distributed training can be done, in torch, it's pretty simple, I don't know about TensorFlow.

File: train.py [hunk 3]
```diff
@@ -163,7 +171,7 @@ def train(
         default="imle_net",
         help="Select the model to train. (imle_net, mousavi, rajpurkar)",
     )
-    parser.add_argument("--batchsize", type=int, default=32, help="Batch size")
+    parser.add_argument("--batchsize", type=int, default=32, help="Batch size") # Reduce the batch size if any memory related problem comes up
```
Comment (likith012) [inline train.py]: there is a help tag to provide info for argparser

Comment (likith012) [reply train.py]: Add the comment into that

**Review by likith012 — CHANGES_REQUESTED** [review]
Work on the comments, if distributed training can't be done, it's alright.

File: train.py (hunk 2 above)
Comment (Srijan221) [reply train.py:156]: I have implemented the distibuted training in torch but not in tensorflow as it requires a good amount of original code to be changed. You can refer this link and let me know if I should implement this as well - https://www.tensorflow.org/tutorials/distribute/keras

File: train.py (hunk 3 above)
Comment (Srijan221) [reply train.py]: Done

**Review by likith012 — APPROVED** [review]
Good work, your changes are approved.

**Pull request merged**